ql/src/java/org/apache/hadoop/hive/ql/exec/tez/TezTask.java

```diff
@@ -268,8 +268,8 @@ public int execute() {
         // Log all the info required to find the various logs for this query
         String dagId = this.dagClient.getDagIdentifierString();
         String appId = this.dagClient.getSessionIdentifierString();
-        LOG.info("HS2 Host: [{}], Query ID: [{}], Dag ID: [{}], DAG App ID: [{}]", ServerUtils.hostname(), queryId,
-            dagId, appId);
+        LOG.info("HS2 Host: [{}], Query ID: [{}], Dag ID: [{}], DAG App ID: [{}], DAG App address: [{}]",
+            ServerUtils.hostname(), queryId, dagId, appId, runtimeContext.getAmHostName());
```

Comment:
can't we use `session.getSession().getAmHost()`?

Reply:
other thing, `Ref<TezSessionState> sessionRef` magic is confusing and redundant

Reply:
makes sense, getAmHost works, I changed to it
also removed an unused TezClient import 

regarding the confusing sessionRef, agree, but I'm not about to touch that magic at the moment, created [HIVE-28877](https://issues.apache.org/jira/browse/HIVE-28877)